### PR TITLE
docs: add upcoming angular conferences

### PR DESCRIPTION
**Pull request by mgechev**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by sonukapoor — APPROVED** [review]

**Review by petebacondarwin — APPROVED** [review]
Apparently there will be ngPoland at the same time as EnterpriseNG.

**Comment by sonukapoor** [comment]
@mgechev CI currently fails to detect PRs if they come from `angular/angular` instead of a fork, which is why the deploy_aio job is failing.

**Comment by mgechev** [comment]
@sonukapoor thanks for the heads up.

**Comment by AndrewKushnir** [comment]
Hi @mgechev, it looks like there is a conflict with the master branch after merging https://github.com/angular/angular/commit/5b337987966940019539fd53385e79337106144d. Could you please update your PR and resolve conflicts? Thank you.

**Comment by mgechev** [comment]
@AndrewKushnir done!

**Comment by AndrewKushnir** [comment]
Thanks @mgechev 👍

File: aio/content/marketing/events.json [hunk 1]
```diff
@@ -1,4 +1,31 @@
...
+  },
+  {
+    "name": "ng-china",
+    "location": "Online",
+    "linkUrl": "https://ng-china.org/",
+    "date": {
+      "start": "2020-11-21",
+      "end": "2020-11-22"
+    }
+  },
```
Comment (gkalpak) [inline aio/content/marketing/events.json:28]: Nit: Consider keeping the events in reverse chronological order for easier scanning (all the rest are ordered like that):

```suggestion
  {
    "name": "ng-china",
    "location": "Online",
    "linkUrl": "https://ng-china.org/",
    "date": {
      "start": "2020-11-21",
      "end": "2020-11-22"
    }
  },
  {
    "name": "EnterpriseNG",
    "location": "Online",
    "linkUrl": "https://www.ng-conf.org/",
    "date": {
      "start": "2020-11-19",
      "end": "2020-11-20"
    }
  },
  {
    "name": "ngrome",
    "location": "Online",
    "linkUrl": "https://ngrome.io/",
    "date": {
      "start": "2020-10-20",
      "end": "2020-10-20"
    }
  },
```

**Review by gkalpak — APPROVED** [review]

**Pull request closed without merging**